library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
+        ** Tachypnea <nowiki> {suggestedTag=Frequency} [Numerical value for cycles / minute. (Source: Beniczky ea 2017, Table 16.)]</nowiki>
+            *** <nowiki># {takesValue, valueClass=numericClass} </nowiki>
+        ** Other-respiration-feature <nowiki> {requireChild} [Source: Beniczky ea 2017, Table 16] </nowiki>
             *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-        ** ECG-feature
-            *** ECG-sinus-rhythm <nowiki> {suggestedTag=Frequency} [Normal rhythm.]</nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** ECG-arrhythmia
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** ECG-asystolia <nowiki>[Add duration (range in seconds) and comments in free text.]</nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** ECG-bradycardia <nowiki> {suggestedTag=Frequency} </nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** ECG-extrasystole
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** ECG-ventricular-premature-depolarization <nowiki> {suggestedTag=Frequency} </nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** ECG-tachycardia <nowiki> {suggestedTag=Frequency} </nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Other-ECG-feature {requireChild}
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-    * EMG-channel-finding <nowiki> {suggestedTag=Finding-significance-to-recording} [EMG channel findings involve distinctive muscular activity patterns and characteristics observed in the Electromyography recording.]</nowiki>
-        ** EMG-muscle-side
-            *** EMG-left-muscle
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** EMG-right-muscle
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** EMG-bilateral-muscle
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
+    * ECG-channel-finding <nowiki> {suggestedTag=Finding-significance-to-recording} [Findings in Electrocardiogram recordings. (Source: Beniczky ea 2017, Section 13, Table 16.)]</nowiki>
+        ** ECG-QT-period <nowiki> {requireChild}[Value and free text. (Source: Beniczky ea 2013, Table 9; Beniczky ea 2017, Table 16.)] </nowiki>
+            *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
+        ** ECG-normal-rhythm <nowiki> {suggestedTag=Frequency} [Normal rhythm. (Source: Beniczky ea 2017, Table 16.)]</nowiki>
+            *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
```

Comment:
I'm not sure what would be wanted for the text here.  Could there be a clarification such as:

"Annotation of any unusual characteristics."